.gitignore

```diff
@@ -1,8 +1,23 @@
-node_modules/
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
-cypress/screenshots/
-cypress/videos/
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
 
-/public/build/
```

Comment:
this probably shouldn't be removed, since the result of the build shouldn't go on git 😃

Reply:
when we do `yarn build`. build folder goes outside of public folder. But yet I will add this line to .gitignore

Reply:
okay, I wasn't aware of that. Then we should add the path to where the build folder goes since it's different on React